RecoLocalCalo/HGCalRecAlgos/interface/ClusterTools.h

```diff
@@ -23,25 +23,39 @@ namespace edm {
 }
 
 namespace hgcal {
-  class ClusterTools {    
+  class ClusterTools {
   public:
+    ClusterTools();
     ClusterTools(const edm::ParameterSet&, edm::ConsumesCollector&);
     ~ClusterTools() {}
 
     void getEvent(const edm::Event&);
     void getEventSetup(const edm::EventSetup&);
-    
+
     float getClusterHadronFraction(const reco::CaloCluster&) const;
 
-    math::XYZPoint getMultiClusterPosition(const reco::HGCalMultiCluster&, double vz = 0.) const;
-    
-    
+    math::XYZPoint getMultiClusterPosition(const reco::HGCalMultiCluster&) const;
+
+    int getLayer(const DetId) const;
+
     double getMultiClusterEnergy(const reco::HGCalMultiCluster&) const;
 
+    // only for EE
+    bool getWidths(const reco::CaloCluster & clus,double & sigmaetaeta, double & sigmaphiphi, double & sigmaetaetalog, double & sigmaphiphilog ) const;
   private:
+
+    std::vector<size_t> sort_by_z(const reco::HGCalMultiCluster&v) const {
+      std::vector<size_t> idx(v.size());
+      std::iota (std::begin(idx), std::end(idx), 0);
```

Comment:
This line broke the IB builds using gcc63 and gcc70. I believe the problem is you need a `#include  <numeric>`.